dashboard/src/components/AppView/AccessURLTable/AccessURLTable.test.tsx

```diff
@@ -178,6 +178,41 @@ context("when the app contains ingresses", () => {
     expect(wrapper.find("a").findWhere(a => a.prop("href") === "http://foo.bar/ready")).toExist();
     expect(wrapper).toMatchSnapshot();
   });
+
+  it("should show the table with available ingresses without anchors if a regex is present in the path", () => {
+    const ingress = {
+      kind: "Ingress",
+      metadata: {
+        name: "foo",
+        selfLink: "/ingresses/foo",
+      },
+      spec: {
+        rules: [
+          {
+            host: "foo.bar",
+            http: {
+              paths: [{ path: "/ready(/|$)(.*)" }],
+            },
+          },
+        ],
+      } as IIngressSpec,
+    } as IResource;
+    const ingressItem = { isFetching: false, item: ingress };
+    const url = ingress.metadata.selfLink;
+    const ingressRefs = [{ name: "svc", getResourceURL: jest.fn(() => url) } as any];
+    const state = { kube: { items: { [url]: ingressItem } } };
+    const store = getStore(state);
+    const wrapper = mountWrapper(
+      store,
+      <AccessURLTable {...defaultProps} ingressRefs={ingressRefs} />,
+    );
+    expect(wrapper.find("Table")).toExist();
+    expect(
+      wrapper.find("a").findWhere(a => a.prop("href") === "http://foo.bar/ready(/|$)(.*)"),
```

Comment:
I would do `expect(wrapper.find("a")).not.toExistst()` so we are sure we are not parsing the URL in a weird way and generate a link.